scarb/src/core/manifest/toml_manifest.rs

```diff
@@ -197,6 +199,7 @@ pub struct TomlPackage {
     pub license_file: Option<MaybeWorkspaceField<Utf8PathBuf>>,
     pub readme: Option<MaybeWorkspaceField<PathOrBool>>,
     pub repository: Option<MaybeWorkspaceField<String>>,
+    pub include: Option<MaybeWorkspaceField<Vec<String>>>,
```

Comment:
this field should be a separate PR

Originally posted in: https://github.com/software-mansion/scarb/pull/1789#discussion_r1872775817